MobileCenter/MobileCenter/Internals/Storage/MSDBStorage.m

```diff
@@ -31,9 +24,9 @@ - (instancetype)init {
   return self;
 }
 
--(void)initTables {
+- (void)initTables {
   NSString *createLogTableQuery = [NSString stringWithFormat:@"create table if not exists %@ (%@ text, %@ text);",
-                                   kMSLogTableName, kMSStorageKeyColumnName, kMSDataColumnName];
+                                                             kMSLogTableName, kMSGroupIDColumnName, kMSDataColumnName];
```

Comment:
There is a PR which changes GroupID -> GroupId. Change it to `kMSGroupIdColumnName`